qemu/tests/qemu_guest_agent.py

```diff
@@ -1949,6 +1949,34 @@ def gagent_check_with_migrate(self, test, params, env):
         error_context.context("Verify if guest agent works.", logging.info)
         self.gagent_verify(self.params, self.vm)
 
+    @error_context.context_aware
+    def gagent_check_user_logoff(self, test, params, env):
+        """
+        Check guest agent status when user is logged out.
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment.
+        """
+        session = self._get_session(params, None)
+        self._open_session_list.append(session)
+
+        error_context.context("Check which user is logged in.", logging.info)
+        user_info = session.cmd_output('query user | findstr /i "Active"')
+        login_user_id = user_info.strip().split()[2]
+
+        error_context.context("Make the user log out.", logging.info)
+        try:
+            session.cmd("logoff %s" % login_user_id)
```

Comment:
@xiagao This case don't need to cover windows guest,right?

Reply:
> @xiagao This case don't need to cover windows guest,right?

it's only cover windows guest. thanks.